packages/demo-app/src/App.tsx

```diff
@@ -45,25 +46,28 @@ export function App(): React.ReactElement {
         <p>Loading RegenApi...</p>
       )}
 
-      <div>
-        <h2>Settings</h2>
-        <label htmlFor="tm">
-          The current Tendermint RPC endpoint we&apos;re using:
-        </label>
-        <input
-          name="tmUrl"
-          value={tmUrl}
-          onChange={({ target: { value } }) => setTmUrl(value)}
-        />
-        <p>
-          Switch to a known public endpoint:
-          <button onClick={() => setTmUrl(DEVNET_NODE_TM_URL)}>
-            Official Regen Devnet node
-          </button>
-          <button onClick={() => setTmUrl(AMAURY_NODE_TM_URL)}>
-            Amaury&apos;s node
-          </button>
-        </p>
+			<div>
```

Comment:
seems like there are some formatting inconsistencies between the code above and the code below (mixture of spaces and tabs for indents)

Reply:
added a rule for 2-space indents

Reply:
it seems like there are still some issues in this file
![image](https://user-images.githubusercontent.com/3149358/151759450-d6c45aae-9745-4587-9345-e145505f89dd.png)

Reply:
thanks. This should be fixed now. It wasn't easy to see in my editor.